vhdl_lang/src/syntax/concurrent_statement.rs

```diff
@@ -52,11 +51,114 @@ pub fn parse_block_statement(
     stream.expect_kind(SemiColon)?;
     Ok(BlockStatement {
         guard_condition,
+        header,
         decl,
         statements,
     })
 }
 
+fn parse_block_header(
+    stream: &mut TokenStream,
+    diagnostics: &mut dyn DiagnosticHandler,
+) -> ParseResult<BlockHeader> {
+    let mut has_port_clause = false;
```

Comment:
Is not has_X redundant with X.is_some()?

Reply:
They are, will update